app/src/main/java/org/mozilla/fenix/home/pocket/PocketRecommendationsHeaderViewHolder.kt

```diff
@@ -0,0 +1,63 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.home.pocket
+
+import android.view.View
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.ComposeView
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.LifecycleOwner
+import androidx.recyclerview.widget.RecyclerView
+import org.mozilla.fenix.R
+import org.mozilla.fenix.compose.ComposeViewHolder
+import org.mozilla.fenix.theme.FirefoxTheme
+
+/**
+ * [RecyclerView.ViewHolder] for displaying the Pocket feature header.
+ *
+ * @param composeView [ComposeView] which will be populated with Jetpack Compose UI content.
+ * @param viewLifecycleOwner [LifecycleOwner] to which this Composable will be tied to.
+ * @param interactor [PocketStoriesInteractor] callback for user interaction.
+ */
+class PocketRecommendationsHeaderViewHolder(
+    composeView: ComposeView,
+    viewLifecycleOwner: LifecycleOwner,
+    private val interactor: PocketStoriesInteractor
+) : ComposeViewHolder(composeView, viewLifecycleOwner) {
+
+    @Composable
+    override fun Content() {
+        val horizontalPadding = with(composeView.resources) {
+            getDimensionPixelSize(R.dimen.home_item_horizontal_margin) / displayMetrics.density
+        }
```

Comment:
I just curious about this piece of code. I am wondering what is the difference between doing this vs other places (which might be actually wrong) that only has `val horizontalPadding = composeView.resources.getDimensionPixelSize(R.dimen.home_item_horizontal_margin)`

Reply:
I was transforming
> density independent pixel -> pixels -> `compose.ui.unit.Dp`
>  xml: 16                      ->  getDimension: 60    ->  divided by density: 16

because dp is what `Modifier.padding` expected and I forgot compose has a `dimensionResource` that would get us that value without the cumbersome transformation. 

In the other cases I see the old `View.padding` api is used that takes pixels as argument. (in the example above 60).

I'll switch to use `composeView.setPadding(..)` for consistency here also (except for in `PocketStoriesViewHolder` which should occupy the full width for the content to have shadows).